strapi/strapi
# chore: add isHostedOnCloud property in NPS request

### What does it do?

The NPS score needs to include information about whether the project is run on Cloud or not.

### Why is it needed?

We want to track feedback specifically coming from the projects run on Cloud.

### How to test it?

Actually, I'm struggling to test this on Cloud.
I tested it only on local:
- run the getstarted project
- make the NPS panel appear
- send the feedback
- check the request sent, it includes the `onCloud` property
